Makefile

```diff
@@ -92,6 +92,8 @@ GOPKGDIR := $(GOPATH)/src/$(PROJECT)
 GOPKGBASEDIR := $(shell dirname "$(GOPKGDIR)")
 GO_FILES := $(shell find . -type f -name '*.go' -not -name '*_test.go')
 
+GINKGO_SKIP_PACKAGES = test/nri
```

Comment:
might be deserving comment, why those are skipped? e.g. to mark that those are e2e style tests, not unit tests?

Reply:
Added a comment explaining the reason for skipping.